other/ethhub-cftc-response.md

```diff
@@ -54,9 +59,12 @@ A user must only wait 1 block confirmation for their transaction to be validated
 
 From a high level, Ethereum is similar to Bitcoin in that it uses a 'blockchain' and uses 'Proof of Work' to achieve consensus and secure the network. Both blockchains have a native digital asset - Ether \(ETH\) for Ethereum and Bitcoin \(BTC\) for Bitcoin.
 
-The Ethereum network is a turing-complete blockchain. This means that it is capable of executing arbitrary code - known as smart contracts. The ability to run code on the Ethereum blockchain means that developers can build applications on top of it. One such application, MakerDAO, allows users to use their ETH as collateral to take out a loan denominated in a USD-pegged stablecoin \(known as DAI\). Applications are all interoperable within the confines of the Ethereum network. Ethereum acts as a common protocol for all decentralized applications built atop, in the way that TCP/IP acts as a common protocol for all public web-based applications. It is not strictly limited to being used as a settlement layer as Bitcoin has become.
+The Ethereum network is a turing-complete blockchain. This means that it is capable of executing arbitrary code - known as smart contracts. The ability to run code on the Ethereum blockchain means that developers can build applications on top of it. Applications are all interoperable within the confines of the Ethereum network. Ethereum acts as a common protocol for all decentralized applications built atop, in the way that TCP/IP acts as a common protocol for all public web-based applications. It is not strictly limited to being used as a settlement layer as Bitcoin has become.
 
-Ethereum 2.0 \(Serenity\) is a substantial upgrade for the Ethereum network being deployed in multiple phases. The main features in this upgrade include a move to a novel Proof of Stake system known as Casper \(for securing the network\), sharding \(for scaling the network\) and eWASM \(a new virtual machine for the network\). These upgrades and new features are developed by a decentralized network of core developers spanning 8 individual Ethereum software client teams, the non-profit Ethereum Foundation, and additional volunteer developers and researchers from around the globe.
```

Comment:
I think it is useful to mention that the core development spans multiple teams in a decentralized way.